crates/fuel-core/src/graphql_api/storage.rs

```diff
@@ -141,6 +142,14 @@ where
         Ok(new_tx_count)
     }
 
+    fn get_tx_count(&self) -> StorageResult<u64> {
+        const TX_COUNT: &str = "total_tx_count";
```

Comment:
It would be nice to move it on one level upper. And reuse inside of the `increase_tx_count`

Reply:
By one level up, do you mean the `Transactional` trait?

Reply:
I mean reuse one constant in two functions=) Define it on file level instead of function level